### ISSUE TITLE
web: color visualizer does not affect `body` element's background

### ISSUE THREAD

**Issue by dbjorge** [microsoft/accessibility-insights-web#4615]
## Describe the bug

Accessibility Insights for Web's Color ad-hoc visualizer, which is intended to turn a page grayscale, does not work against background colors/images applied to a page's body element.

**To Reproduce**

1. Go to any page that uses a `background-color` or `background-image` style on its `body` element. For example, https://dequeuniversity.com/demo/mars/ or a local page using the repro in the next section of this issue
2. Active the Color ad-hoc tool
3. Observe that the page's background color does not change

## Repro example

```html
<html>
<head>
<style>
body {
    /* Either one of these repros */
    background-color: #ffaaaa;
    background-image: linear-gradient(#ffaaaa, #aaffaa)
}
</style>
</head>
<body>
<p>some text</p>
</body>
</html>
```

## Expected behavior

The entire page, including the body background, should be filtered to grayscale

## Screenshots

![screenshot showing repro page with colorful background despite active color ad-hoc tool](https://user-images.githubusercontent.com/376284/130868967-b9f04a83-728e-4b58-b79a-0f21540858c2.png)

## Context (please complete the following information)

- OS Name & Version: Windows 21H1 (OS Build 19043.1165)
- AI-Web Version & Environment: Repros on both 2.27.0 (prod) and 2.28.0 (insider)
- Browser Version: Microsoft Edge Version 92.0.902.78 (Official build) (64-bit)
- Target Page: see repro example

## Are you willing to submit a PR?

Yes. I believe the fix is to apply our grayscale filter style to the HTML element, rather than the body element.

## Did you search for similar existing issues?

yes

## Additional context

n/a

**Comment by peterdur** [comment]
This bug feels very familiar, but I've had no luck finding an older instance with a reason that this couldn't be fixed.

Validated that applying the filter on the html element rather than on the body element appears to fix this.

**Comment by ferBonnin** [comment]
reviewed with Peter. To consider that this is also used in Assessment
